collections/_article/dev-snapshot-godot-4-2-beta-1.md

```diff
@@ -0,0 +1,382 @@
+
+To close things off, myself, [Yuri Sizov](https://github.com/YuriSizov), has worked out several annoyances and issues in the texture region editor, making sure it works consistently with all supported resources, such as styleboxes and atlases ([GH-80435](https://github.com/godotengine/godot/pull/80435)).
+
+#### Asset library
+
+A couple of useful changes made their way into the AssetLib tab. Now, as you install an addon or an asset, be it from a file or from the search results, you can specify a different installation folder and also choose to skip the root folder in the asset archive typically generated by GitHub ([GH-81358](https://github.com/godotengine/godot/pull/81358), [GH-81620](https://github.com/godotengine/godot/pull/81620) by [Yuri Sizov](https://github.com/YuriSizov)).
+
+The search results themselves have been improved, solving a problem with overflowing content and adjusting navigation buttons ([GH-80555](https://github.com/godotengine/godot/pull/80555) by [Dalton Lang](https://github.com/GrammAcc)).
+
+#### Project manager
+
+The project manager is the first thing that users see when they begin their Godot journey, and as such it is both in a constant need to be improved and must be treated with care. As a result of that dilemma, the work submitted by [starry-abyss](https://github.com/starry-abyss) two years ago only now becomes available with the upcoming release of Godot 4.2.
+
+They've looked into the UX of the project manager and decided to improve the general arrangement of key controls, as well as the project import workflow ([GH-50674](https://github.com/godotengine/godot/pull/50674), [GH-51478](https://github.com/godotengine/godot/pull/51478)). Make sure to give us your feedback on these changes!
+
+![A look at the updated layout of the project manager](/storage/blog/godot-4-2-beta/project-manager-layout.webp)
+
+#### Editor plugins
+
+In more technical news, the editor-specific scripting API, relevant for plugins and tool scripts, has seen a few changes.
+
+A lot of editor functionality is exposed, directly or indirectly, via the `EditorInterface` class. It exists as a single instance, however in previous versions of Godot accessing it involved getting a reference through `EditorPlugin` or `EditorScript`. In case of tool scripts this meant using hacks creating useless instances of either class just to get the access. Now, `EditorInterface` is just a singleton, available only in the editor ([GH-75694](https://github.com/godotengine/godot/pull/75694) by [Yuri Sizov](https://github.com/YuriSizov)).
```

Comment:
Avoids redundancy with the previous sentence.
```suggestion
A lot of editor functionality is exposed, directly or indirectly, via the `EditorInterface` class. It exists as a single instance, however in previous versions of Godot accessing it involved getting a reference through `EditorPlugin` or `EditorScript`. In case of tool scripts this meant using hacks creating useless instances of either class just to get the access. Now, `EditorInterface` is a singleton, available only in the editor ([GH-75694](https://github.com/godotengine/godot/pull/75694) by [Yuri Sizov](https://github.com/YuriSizov)).
```